CONTRIBUTING.md

```diff
@@ -1,12 +1,47 @@
-## Contributing
+# Contributing
 
-Pull requests for bug fixes are always welcome!
+🎉 Welcome to the OpenTelemetry Java Contrib Repository! 🎉
+
+## Introduction
+
+This repository focuses on providing tools and utilities for Java-based observability, such as remote JMX metric gathering and reporting. We’re excited to have you here! Whether you’re fixing a bug, adding a feature, or suggesting an idea, your contributions are invaluable.
 
-Before submitting new features or changes to current functionality, it is recommended to first
+-Before submitting new features or changes to current functionality, it is recommended to first
 [open an issue](https://github.com/open-telemetry/opentelemetry-java-contrib/issues/new)
 and discuss your ideas or propose the changes you wish to make.
 
-### Building
+-Questions? Ask in the OpenTelemetry [java channel](https://cloud-native.slack.com/archives/C014L2KCTE3)
+
+Pull requests for bug fixes are always welcome!
+
+## Pre-requisites
+
+To work with this repository, ensure you have:
+
+Tools:
+
+-JDK 11+
+
+-Java 17 or higher
+
+-Gradle 7+
```

Comment:
we use the gradle wrapper, so I don't think gradle should be listed as a prerequisite?